src/cpp/server/residency/platform/durable_file_adapter_windows.cpp

```
@@ -0,0 +1,1439 @@
+        }
+        return captured;
+    }
+    std::optional<DurablePreflightTestFault> preflight_test_fault_;
+    bool preflight_test_fault_taken_ = false;
+#endif
+};
+
+} // namespace
+
+std::unique_ptr<DurableFileAdapter>
+make_platform_durable_file_adapter(const std::filesystem::path &directory) {
```

Cppcheck (2.21.0)

[style] The function 'make_platform_durable_file_adapter' is never used.

(unusedFunction)